### ISSUE TITLE
NI NO KUNI [BLES01555] no more playable - regression due to #4553 (0.0.5-6722)

### ISSUE THREAD

**Issue by psennermann** [RPCS3/rpcs3#4589]
PR #4553 (0.0.5-6722) causes the game NiNoKuni to get stuck/hang after beginning a new game and choosing difficulty level or loading a saved game.
Log can be found here: #4565

P.S. just a personal humble thought, but is it worth merging PR that fix "secondary" games at the cost of breaking "mass-market" games?

**Comment by Illynir** [comment]
Confirmed regression.

**Comment by Relsig** [comment]
I can also confirm this.

P.s. In regards to the aside in the op
It's not about breaking mass market games or fixing secondary games it's about emulating the PS3 as accurately and efficiently as possible.

Sometimes a piece of code that makes a game work is wrong and gets fixed, which breaks said game until a proper fix for that case is created. Other times a commit doesn't function exactly as intended and introduces regressions. There are many other potential reasons for regressions, both intentional and accidental, but I won't bore you with them.

Kd-11, Nekotekina, and all the other developers submitting code are working hard to make rpcs3 as accurate and efficient as possible and, honestly, I'm constantly amazed by their pace.

I'm sure they appreciate your help testing for regressions and your consideration and understanding when problems do arise.

**Comment by fboudra** [comment]
I can confirm the issue as well. For the record, here's a link to build 0.0.5-6718 which is working with ni no kuni: https://ci.appveyor.com/project/rpcs3/rpcs3/build/0.0.5-6718/artifacts

**Comment by psennermann** [comment]
Yes, I understand, but in this particular case, with so many important games that got broken, I think it would have been better first to revert and then to find calmly an alternative solution for the broken code, instead of forcing people to stick with older build losing in this way potential testers for newer releses.
But that's only my opinion and for sure doesn't lessen the work of the developers that is absolutely astonishing.

**Comment by Relsig** [comment]
@psennermann

TL;DR, probably won't happen that way for a long time.

This is due to one of the many other reasons i didn't want to bore you with above -> one game may not highlight the issue in a way that another does and the more ways the issue is described the more likely they are to spot the cause and thus fix the issue and then reap the benefits of increased accuracy and efficiency.

In response to the above you might say then 'leaving the pr open would do the same' but then other fixes that depend on that code would end up delayed and other developers would have to keep the second branch in mind when working with the old code and in general it just leads to messyness that can easily be avoided.

This is one reason why programs have 'stable' and 'nightly' builds. Rpcs3 is still pre-alpha software and hasn't reached a milestone of stability yet, let alone feature-completeness this is why the version is 0.0.5 instead of '1.x.x'. Once it is stable and feature complete enough there will most likely be stable and nightly builds.

**Comment by psennermann** [comment]
Closing since issue has been fixed by 0.0.5-7069

**Comment by AniLeo** [comment]
Fixed by #4816
